DOCS-9278-email-send-kt (#1112)
mulesoft/docs-connectors: email/1.3/modules/ROOT/pages/email-trigger.adoc

= Trigger a Flow When Receiving a New Email
:page-aliases: connectors::email/email-trigger.adoc

> Anypoint Connector for Email (Email Connector) provides the *On New Email - IMAP* and *On New Email - POP3* source listeners that trigger a flow by retrieving an email folder from a mailbox for all the unread emails. A new message is then generated for each email found. The following examples illustrate how to configure these source parameters to identify a new email when triggering a flow.

Reviewer: Suggestions: "trigger" > "initialize" 
"all the" > "all of"

> Anypoint Connector for Email (Email Connector) provides the *On New Email - IMAP* and *On New Email - POP3* source listeners that initiates a flow by retrieving an email folder from a mailbox for all of the unread emails. A new message is then generated for each email found. The following examples illustrate how to configure these source parameters to identify a new email when initiating a flow.

Reviewer 2: So, the connector includes two source listeners, one for IMAP email and one for POP3.
The source listeners initiate a flow by retrieving [an email message or a folder of messages?] from the mailbox of unread email. [Do we really mean that the listeners retrieve ALL unread email and create a message for each?]
"These source parameters" refers to ...the source listeners On New Email - IMAP and POP3?
And they identify a new [unread?] email?
Maybe we mean
```suggestion
The following examples illustrate how you configure the connector source listeners *On New Email - IMAP* and *On New Email - POP3* to identify and retrieve unread email messages, initiating a flow.
```
???
